### PR TITLE
fix: invalid textfield behaviour with number value

### LINKED ISSUE #2359: [Bug] [v-text-field] interpretes negative numbers as NaN
### Versions and Environment

Vuetify: `v0.17.0-beta1`
Vue: `v2.5.2`
Browsers: `Chrome,Firefox,Safari`
OS: `macOS`

### Expected Behavior
<v-text-field> should allow the entry of negative numbers.

### Actual Behavior

But with v0.17.0-beta.1, as soon as you enter a `minus`, the input will be replaced with a literal string of `NaN`.

### Reproduction Link

https://codepen.io/myleslee/pen/xPxBQY

### Other comments

The issue cannot be reproduced on `v0.16.9` though.

### PR DESCRIPTION
**Pull request by jacekkarczmarczyk**
fixes #2359

### CODE REVIEW

**Comment by KaelWD** [comment]
@azaars Does this break anything from #2099?

**Comment by azaars** [comment]
Based on the discussion on discord, this PR is suffice and it won't break anything from #2099 

@jacekkarczmarczyk No need to do `this.lazyValue = String(val)`

File: src/components/VTextField/VTextField.js [hunk 1]
```diff
@@ -89,12 +89,10 @@ export default {
       },
       set (val) {
         if (this.mask) {
-          const value = this.unmaskText(this.maskText(this.unmaskText(val)))
-          this.lazyValue = typeof val === 'number' ? +value : value
+          this.lazyValue = this.unmaskText(this.maskText(this.unmaskText(val)))
```
Comment (johnleider) [inline src/components/VTextField/VTextField.js]: O_o

**Comment by johnleider** [comment]
I went to test this on DEV and the issue does not occur, maybe this can be closed.

**Pull request merged**